docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -2053,303 +2053,285 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Action:**
 
-  TODO
+  Check any missing argument or unsupported argument.
+
 
 
+### ZWED0154W
 
-### ZWED0158W":"%s
+  WARNING: Unrecognized command: _args_
 
   **Reason:**
 
-  TODO
+  Throws warning error _args_ when command is Unrecognized.
 
   **Action:**
 
-  TODO
+  Check the command once again.
 
 
 
-### ZWED0159W":"Plugin (%s) loading failed. Message: "%s" Successful: %s% (%s/%s) Attempted: %s% (%s/%s)
+### ZWED0155W
+
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for detail to debug.
 
 
 
-### ZWED0160W":"Master cluster not found to save storage for id: %s, value: %s
+### ZWED0156W
+
+  1 function initLoggerMessages - ERROR - _Error_
 
   **Reason:**
 
-  TODO
+  When we encountered an _Error_ ,then 1 function created a logger file with specified message.
 
   **Action:**
 
-  TODO
+  Go through the logger file for detail _Error_ to debug.
+
 
 
+### ZWED0157W
 
-### ZWED0161W":"Master cluster not found to save storage
+  2 function initLoggerMessages - ERROR - _Error_.
 
   **Reason:**
 
-  TODO
+  When we encountered an _Error_ ,then 2 function created a logger file with specified message.
 
   **Action:**
 
-  TODO
+  Go through the logger file for detail _Error_ to debug.
+
 
 
+### ZWED0158W
 
-### ZWED0162W":"Master cluster not found to add storage
+   _ErrorMsg_
 
   **Reason:**
 
-  TODO
+  Server throws 500 code with _ErrorMsg_.
 
   **Action:**
 
-  TODO
+  Go through the _ErrorMsg_ for detail to debug.
+
 
 
+### ZWED0159W
 
-### ZWED0163W":"Master cluster was not found to delete storage for id: %s
+  Plugin (_PluginIdentifier_) loading failed. Message: "_pluginsTotal_" Successful: _pluginsLoaded_% (_pluginsLoaded_/_eventCount_) Attempted: _pluginCount_% (_pluginCount_/_eventCount_)
 
   **Reason:**
 
-  TODO
+  Plugin with _pluginId_ loaded failed. The server attempted to load a total of _pluginsTotal_ with _pluginsLoaded_ plugins already successfully loaded.
 
   **Action:**
 
-  TODO
+  Review _errorMessage_ to see if there is something you can do to fix the error. You may need to contact the plugin developer to find a solution. If you do not need this plugin, it is OK to continue.
 
 
 
-### ZWED0165W":"Unable to retrieve storage data from cluster promise %s
+### ZWED0166W
+
+  Error updating the storage: _Error_
 
   **Reason:**
 
-  TODO
+  Throws warning _Error_ when it faced error while updating the storage.
 
   **Action:**
 
-  TODO
+  Contact support if _Error_ is not clear.
+
 
 
+### ZWED0167W
 
-### ZWED0166W":"Error updating the storage: %s
+  Error adding to the storage: _storageTimeout_
 
   **Reason:**
 
-  TODO
+  Throws _storageTimeout_ error while adding to the storage.
 
   **Action:**
 
-  TODO
+  You may change the value of _storageTimeout_ ms by setting 'node.cluster.storageTimeout' within the config.
```

Comment:
This is totally wrong unfortunately! Please review what ZWED0167W says
You wrote what ZWED0144E is about